libvisual/CMakeLists.txt

```diff
@@ -102,6 +103,13 @@ ELSEIF(SIZEOF_SIZE_T EQUAL SIZEOF_INT)
 ELSEIF(SIZEOF_SIZE_T EQUAL SIZEOF_LONG)
   SET(VISUAL_SIZE_TYPE long)
   SET(VISUAL_SIZE_T_FORMAT "lu")
+ELSEIF(SIZEOF_SIZE_T EQUAL SIZEOF_LONG_LONG)
+  SET(VISUAL_SIZE_TYPE "long long")
+  IF(WIN32)
```

Comment:
I think this needs to be `MSVC` rather than `WIN32`?

Reply:
No since this type also applies to mingw-w64 gcc

Reply:
Seems like you're right:
```console
# cat main.c 
#include <stdio.h>

int main() {
    const unsigned long long llu = 0x1234567812345678lu;
    printf("sizeof(long long): %u\n", sizeof(long long));
    printf("         expected: 1311768465173141112\n");
    printf("             I64u: %I64u\n", llu);
    return 0;
}

# i686-w64-mingw32-gcc -Wall -Wextra main.c && wine ./a.exe 
sizeof(long long): 8
         expected: 1311768465173141112
             I64u: 1311768465173141112
```